src/SecureHeaders.php

```diff
@@ -957,15 +938,15 @@ private function importHeaders()
         $this->importStarted = true;
         # grab any headers that were already set and, if any,
         # add these to our internal header list
-        $this->headerBag = $this->httpAdapter->getHeaders();
+        $this->headers = $this->httpAdapter->getHeaders();
 
-        foreach ($this->headerBag->get() as $header)
+        foreach ($this->headers->get() as $header)
         {
             $this->addHeader($header->getName(), $header->getValue());
         }
 
         # delete them (we'll set them again later)
-        $this->headerBag->removeAll();
+        $this->headers->removeAll();
```

Comment:
This line no longer needed (since the `HeaderBag` now *is* the internal header list, no need to clear it out to avoid sync issues 😄 )